src/index.js

```diff
@@ -64,6 +64,7 @@
   let blueprint;
   let packageUrl;
 
+  debugger;
```

Comment:
🔪

Reply:
I couldn't get this branch working locally :-\
I've abandoned this for now, but to anyone else who needs local paths: feel free to take all this (or do better than I did :D )